src/anm/utils.js

```diff
@@ -199,6 +197,13 @@ function fit_rects(pw, ph, aw, ah) {
     } else return [ 1, [ 0, 0, aw, ah ] ];
 }
 
+function removeElement(array, element) {
```

Comment:
Ok, [I found why it's bad](http://www.smashingmagazine.com/2012/11/05/writing-fast-memory-efficient-javascript/), thank you, so yeah, `null`-ing is the proper way here (since we need to get `if (modifers[id])` fail to `false`).